From f5ccd335672ce751cfa5e84fee8c158c0d0ca833 Mon Sep 17 00:00:00 2001
From: Fan Zhang <zhfan@google.com>
Date: Thu, 6 Sep 2018 12:21:30 -0700
Subject: [PATCH 34/71] Move safety & regulatory manual closer to legal
 settings

Bug: 113788849
Test: visual
Change-Id: Ie9a706bad9a4c65ff06c2350171ad55f46dd8386
Merged-In: Ie9a706bad9a4c65ff06c2350171ad55f46dd8386
---
 res/xml/my_device_info.xml | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/res/xml/my_device_info.xml b/res/xml/my_device_info.xml
index 3dac8b879f..3d499ab460 100644
--- a/res/xml/my_device_info.xml
+++ b/res/xml/my_device_info.xml
@@ -71,10 +71,18 @@
         <intent android:action="android.settings.SHOW_REGULATORY_INFO"/>
     </Preference>
 
+    <!-- Safety & regulatory manual -->
+    <Preference
+        android:key="safety_info"
+        android:order="7"
+        android:title="@string/safety_and_regulatory_info">
+        <intent android:action="android.settings.SHOW_SAFETY_AND_REGULATORY_INFO"/>
+    </Preference>
+
     <!-- SIM status -->
     <Preference
         android:key="sim_status"
-        android:order="7"
+        android:order="8"
         android:title="@string/sim_status_title"
         settings:keywords="@string/keywords_sim_status"
         android:summary="@string/summary_placeholder"
@@ -129,14 +137,6 @@
         android:summary="@string/summary_placeholder"
         android:selectable="false"/>
 
-    <!-- Safety & regulatory manual -->
-    <Preference
-        android:key="safety_info"
-        android:order="39"
-        android:title="@string/safety_and_regulatory_info">
-        <intent android:action="android.settings.SHOW_SAFETY_AND_REGULATORY_INFO"/>
-    </Preference>
-
     <!-- Manual -->
     <Preference
         android:key="manual"
@@ -166,4 +166,4 @@
         android:summary="@string/summary_placeholder"
         settings:allowDividerAbove="true"/>
 
-</PreferenceScreen>
\ No newline at end of file
+</PreferenceScreen>
-- 
2.17.1

